agent/xds/testdata/listeners/terminating-gateway-custom-and-tagged-addresses.envoy-1-12-x.golden

```diff
@@ -183,18 +183,33 @@
           },
           "filters": [
             {
-              "name": "envoy.filters.network.rbac",
+              "name": "envoy.http_connection_manager",
```

Comment:
I think this is a side effect of an earlier change in a `testing.go` file to switch a default from tcp->http. Maybe it would make more sense to revert that and make a new test helper for http only.